ui-admin/src/util/tableUtils.tsx

```diff
@@ -45,25 +56,76 @@ function Filter<A>({
 }: {
   column: Column<A>
 }) {
-  const columnFilterValue = column.getFilterValue()
+  return column.columnDef.meta?.columnType === 'boolean' ? (
+    SelectFilter({ column })
+  ) : (
+    //Defaults to the text search filter
+    SearchFilter({ column })
+  )
+}
+
+/**
+ * returns a SelectFilter to handle fields filtered by dropdown
+ * */
+function SelectFilter<A>({
+  column
+}: {
+  column: Column<A>
+}) {
+  const [selectedValue, setSelectedValue] = useState({})
 
+  return <div>
+    <Select
+      options={column.columnDef.meta?.filterOptions || []}
+      styles={{
+        control: baseStyles => ({
+          ...baseStyles,
+          width: 200,
+          fontWeight: 'normal'
+        }),
+        menu: baseStyles => ({
+          ...baseStyles,
+          fontWeight: 'normal'
+        })
+      }}
+      value={selectedValue}
+      // @ts-ignore
```

Comment:
> Is it because Line 75 doesn't have a type declaration so it's inferring type based off of the default state?

I would guess so. My guess is that Select has a type parameter for options/value that it's inferring from the value prop. And since `selectedValue` is initialized with `{}`, it thinks that any value will be an empty object.

Does something like this fix it? Note that this would use `undefined` to represent "no selection" and there are probably places where that undefined would have to be handled.
```ts
const [selectedValue, setSelectedValue] = useState<{ value: boolean, label: string }>()
```

Reply:
Is it possible to get rid of this ts-ignore by adding a type parameter to `useState`?

Reply:
Unfortunately not. I did add the type parameter in 9f39258c. That commit also better handles the null case and gets rid of the `Any` Select option, which complicated the typing. Instead it uses the built-in `isClearable` option.